packages/core/strapi/bin/strapi.js

```diff
@@ -69,10 +68,10 @@ const getLocalScript =
 
 // option to exclude types of data for the export, import, and transfer commands
 // TODO: validate these inputs. Hopefully here, but worst case it may require adding a hook on each command
-const excludeOption = new Option(
-  '--exclude <data,to,exclude>',
-  'Comma-separated list of data to exclude (files [localMediaFiles, providerMediaFiles], content [entities, links], schema, configuration)' // ['webhooks', 'content', 'localmedia', 'providermedia', 'relations']
-).argParser(parseInputList);
+// const excludeOption = new Option(
```

Comment:
Same as below, we can add it back later based on the git history, I don't think it's worth keeping this as a comment.